numba/targets/arrayobj.py

```diff
@@ -1706,6 +1706,60 @@ def scalar_sinc_impl(val):
                                    locals=dict(c=scalar_dtype))
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
+@builtin
+@implement(numpy.angle, types.Kind(types.Number))
+def scalar_angle(context, builder, sig, args):
+    def scalar_angle_impl(val):
+          return numpy.arctan2(val.imag, val.real)
+    res = context.compile_internal(builder, scalar_angle_impl,
+                                      sig, args)
+    return impl_ret_untracked(context, builder, sig.return_type, res)
+
+@builtin
+@implement(numpy.angle, types.Kind(types.Number), types.Kind(types.Boolean))
+def scalar_angle_kwarg(context, builder, sig, args):
+    def scalar_angle_impl(val, deg=False):
+        if(deg):
+            scal = 180/numpy.pi
+            return numpy.arctan2(val.imag, val.real) * scal
+        else:
+            return numpy.arctan2(val.imag, val.real)
+    res = context.compile_internal(builder, scalar_angle_impl,
+                                      sig, args)
+    return impl_ret_untracked(context, builder, sig.return_type, res)
+
+@builtin
+@implement(numpy.angle, types.Kind(types.Array))
+def array_angle(context, builder, sig, args):
+    arg=sig.args[0]
```

Comment:
PEP 8 mandates spaces around assignment and operators, so `arg = sig.args[0]` it should be ;-)